aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -136,11 +180,31 @@ module aptos_framework::coin {
     }
 
     #[event]
+    /// Module event emitted when some amount of a coin is withdrawn from an account.
     struct Withdraw<phantom CoinType> has drop, store {
         account: address,
         amount: u64,
     }
 
+    #[event]
+    /// Module event emitted when the event handles related to coin store is deleted.
+    struct CoinEventHandleDeletion has drop, store {
+        event_handle_creation_address: address,
+        deleted_deposit_event_handle_creation_number: u64,
+        deleted_withdraw_event_handle_creation_number: u64,
+    }
+
+    #[event]
+    /// Module event emitted when a new pair of coin and fungible asset is created.
+    struct PairCreation has drop, store {
+        coin_type: TypeInfo,
+        fungible_asset_metadata_address: address,
+    }
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// The flag the existence of which indicates the primary fungible store is created by the migration from CoinStore.
+    struct MigrationFlag has key {}
```

Comment:
My thought was I am truly not sure whether we will spend time cleaning this so it is better in the same RG otherwise we will have another RG created for each (account, coin)... That's a huge number imho.